samples/node/pub_sub_electron_node/settings.ts

```diff
@@ -0,0 +1,4 @@
+export const key_file_path = "<private key file path>";
+export const cert_file_path = "<certificate file path>";
```

Comment:
Nitpick: I'd ddd a comment here saying if you comment-out/remove `key_file_path` and `cert_file_path` the sample will use Websockets instead of mTLS. If left at it's default, the sample will always use mTLS because it will not be null. (line `33` in `preload_pubsub5.ts`)